site_config/docs.js

```diff
@@ -33,10 +33,6 @@ export default {
             title: 'Backend Development',
             link: '/en-us/docs/developer_guide/backend-development.html',
           },
-          {
-            title: 'Plugin Development',
-            link: '/en-us/docs/developer_guide/plugin-development.html',
-          },
```

Comment:
why remove this link?

Reply:
The plugin development part has no english translation, if config this link, the 404 page will show.